app/config.py

```diff
@@ -25,11 +25,18 @@ class ProductionConfig(Config):
     ENV = 'production'
     MONGO_HOST = os.environ.get('DB_HOSTNAME') or ''
     MONGO_PORT = int(os.environ.get('DB_PORT') or 27017)
-    MONGO_DBNAME = "tdctl"
-    MONGO_URI = "mongodb://%s:%s/%s" % (MONGO_HOST, MONGO_PORT, MONGO_DBNAME)
+    MONGO_DBNAME = "production_db"
```

Comment:
I saw that there is `${DATABASE}` in docker-compose for production that is passed in to the mongodb container. Is this one matching with that? If that is the case we should reuse the env-variable and pass it to the container for the backend as well so that we only have one source of truth

Reply:
right, forgot to use the env variable